File: unimplemented_commands/cookbook.js
// Description
//   Returns a link to the UQCS student-compiled cookbook (full of tasty treats).
// Commands
//   `!cookbook` - returns the URL of the UQCS student-compiled cookbook (pdf).
//

module.exports = function (robot) {
  robot.respond(/!?cookbook/i, function (res) {
    res.send("https://github.com/UQComputingSociety/cookbook");
  });
};

File: uqcsbot/scripts/cookbook.py
from uqcsbot import bot, Command


@bot.on_command("cookbook")
def handle_cookbook(command: Command):
    '''
    `!cookbook` - Returns the URL of the UQCS student-compiled cookbook (pdf).
    '''
    bot.post_message(command.channel, "https://github.com/UQComputingSociety/cookbook")
